### ISSUE TITLE
Add feldspar IRSL global Standardized Growth Curves (Li et al.,2015) to calc_gSGC()

### ISSUE THREAD

**Issue by hjgray10** [R-Lum/Luminescence#98]
Hi all,

I was hoping to request that calc_gSGC() include functionality for IRSL of feldspar per:

Li, B., Roberts, R. G., Jacobs, Z., Li, S. H., & Guo, Y. J. (2015). Construction of a ‘global standardised growth curve’(gSGC) for infrared stimulated luminescence dating of K-feldspar. Quaternary Geochronology, 27, 119-130.

There is currently functionality for quartz OSL, but I was wondering if feldspar IRSL functionality could be added for a future update. I feel that it largely involves adding in an equation and the parameters from the above citation to the current code, but I am unsure of the inner workings of calc_gSGC(). 

Cheers and thanks!

Harrison

**Comment by RLumSK** [comment]
@hjgray10 Is the integration ok? Means, can I close this issue?

**Comment by hjgray10** [comment]
I think so. I was not able to obtain new data to test the function any further. It doesn't look like there is data in Li et al. 2015 that we could use to test against either.

**Comment by RLumSK** [comment]
@hjgray10 No problem. Would you mind double-checking the documentation? If this is also ok, we can ship the function with the upcoming release.

**Comment by hjgray10** [comment]
Sorry to be an idiot – Where is the documentation?

**Comment by RLumSK** [comment]
@hjgray10 Don't worry, it is indeed not obvious because there are two files: 

1. **The R file**:  https://github.com/R-Lum/Luminescence/blob/dev_0.9.x/R/calc_gSGC_feldspar.R
This is the R code file with, what we call, inline documentation. Everything after `#'` is part of the documentation. 

https://github.com/R-Lum/Luminescence/blob/e9cccd0d02053c9a0e5afbe7873ec726f5cdcfc3/R/calc_gSGC_feldspar.R#L1-L67

2. **The Rd-file**: From that file (or code), the documentation is created in https://github.com/R-Lum/Luminescence/blob/dev_0.9.x/man/calc_gSGC_feldspar.Rd
This is what the user sees when calling the manual page of the function, however, in our case *never* touch this file, because it will be overwritten automatically. 

In other words, in order to check and modify the documentation, you have to modify the lines in https://github.com/R-Lum/Luminescence/blob/dev_0.9.x/R/calc_gSGC_feldspar.R

If you need further assistance, please just write again, no problem!

**Comment by RLumSK** [comment]
I guess we have it. I'll close this issue, please feel free to reopen it in case the work was not done.
